src/main/java/com/checkmarx/sdk/utils/scanner/client/ScaClientHelper.java

```diff
@@ -623,15 +623,25 @@ public String createRiskManagementProject(String name) throws IOException {
     }
 
     private void determineProjectTeam(CreateProjectRequest request) {
-        String team = scaProperties.getTeamForNewProjects();
+        String team = config.getScaConfig().getTeam();
+
         if (StringUtils.isNotEmpty(team)) {
-            log.info("Assigning SCA project with team: {}", team);
-            request.setAssignedTeams(Collections.singletonList(team));
+            setTeam(request, team);
         } else {
-            request.setAssignedTeams(null);
+            team = scaProperties.getTeam();
+            if (StringUtils.isNotEmpty(team)) {
+                setTeam(request, team);
+            } else {
+                request.setAssignedTeams(null);
+            }
         }
     }
 
+    private void setTeam(CreateProjectRequest request, String team) {
+        log.info("Assigning SCA project with team: {}", team);
+        request.setAssignedTeams(Collections.singletonList(team));
```

Comment:
If user provides team A here, and the SCA project already exists with team B, what will happen with the project's teams?

Reply:
Team assignment is relevant only on new project creation path. If project already exists team cannot be modified or changed.

Reply:
Ok, so the meaning of scaProperties.getTeam() is still "team for new projects" :)
If the field name must be "team", let's at least add a relevant comment about this field behavior.

Reply:
I added a comment